### ISSUE TITLE
Package is very large

### ISSUE THREAD

**Issue by wch** [r-lib/crayon#69]
I just noticed this:

```
trying URL 'https://cloud.r-project.org/src/contrib/crayon_1.3.4.tar.gz'
Content type 'application/x-gzip' length 658694 bytes (643 KB)
==================================================
downloaded 643 KB
```

It looks like most of the space is taken by images in inst/ which appear to be just there for a readme. Is it possible to move them somewhere else, or at least run optipng on them so they're smaller?

**Comment by mcol** [comment]
Just by compressing the largest image (ANSI-256-OSX.png):
```
                 bytes    diff
original        467885        -
pngcrush        301777   166108  35.5%
optipng         278051   189834  40.5%
```

**Comment by brodieG** [comment]
It may be worth considering using `fansi` to generate the HTML equivalents.  I realize it's kind of cheating, but why not, e.g. as [shown in this tweet](https://twitter.com/BrodieGaslam/status/1029059002225721344).

**Comment by brodieG** [comment]
Actually, I just tried this with show_ansi_colors and the HTML file ends up at 216KB even tar-zipped, so maybe that isn't that helpful.

**Comment by brodieG** [comment]
Ok, so the reason it was so large is because that actually includes jquery and other stuff from `html_document`.  With `html_vignette` it is 25KB uncompressed:

<img width="570" alt="screen shot 2018-08-13 at 1 54 05 pm" src="https://user-images.githubusercontent.com/6105908/44048879-a7a3b9b2-9f00-11e8-961f-fca9bad1d837.png">

**Comment by gaborcsardi** [comment]
Fixed in 4a7436906f2758d1ee27d6d49bca96bf232974df
